javalib/src/main/scala/java/lang/management/MemoryMXBean.scala

```diff
@@ -0,0 +1,42 @@
+package java.lang.management
+
+import scala.scalanative.runtime.GC
+
+trait MemoryMXBean {
+
+  /** Returns the current runtime memory usage.
+   *
+   *  @note
+   *    the committed memory is always -1
+   *
+   *  @note
+   *    the memory usage values may differ between invocations
+   */
+  def getHeapMemoryUsage(): MemoryUsage
+
+  /** Runs the garbage collector.
+   *
+   *  A shortcut for `System.gc`.
+   */
+  def gc(): Unit
+}
+
+object MemoryMXBean {
+
+  private[management] def apply(): MemoryMXBean =
+    new Impl
+
+  private class Impl extends MemoryMXBean {
+    def getHeapMemoryUsage(): MemoryUsage =
+      new MemoryUsage(
+        GC.getInitHeapSize().toLong,
+        GC.getUsedHeapSize().toLong,
+        -1L,
```

Comment:
I didn't find a simple way to get (calculate) the committed memory across all garbage collectors.

Reply:
This one might be more difficult to track. On Windows we do manually commit memory provided by VirtualAlloc, but in the case of Unix we typically only reserve a maximal chunk of memory using mmap and committing is done automatically by the system on the first usage of every page. Access to this memory is only limited by Immix/Commix Heap blocks limit. 
I guess the best choice would be to fill it with usedHeap which is still correct regarding Javadocs: `committed will always be greater than or equal to used`

Reply:
Thanks for the explanations! Let's fill committed memory with the amount of used heap.